client/state.go

```diff
@@ -0,0 +1,88 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+
+	v1 "github.com/dapr/go-sdk/dapr/proto/common/v1"
+	pb "github.com/dapr/go-sdk/dapr/proto/runtime/v1"
+	"github.com/pkg/errors"
+)
+
+// SaveState is the message to save multiple states into state store
+func (c *Client) SaveState(ctx context.Context, store, key string, in []byte) error {
```

Comment:
Need to add ConsistencyMode, ConcurrencyMode & RetryOptions

Reply:
provides additional methods where these value can be specified